Modules/readline.c

```diff
@@ -212,6 +212,9 @@ readline_read_init_file_impl(PyObject *module, PyObject *filename_obj)
         errno = rl_read_init_file(NULL);
     if (errno)
         return PyErr_SetFromErrno(PyExc_OSError);
+    if (!using_libedit_emulation) {
+        rl_variable_bind ("enable-bracketed-paste", "off");
+    }
```

Comment:
I'm not sure that it's a good idea to explicitly ignore the user configuration, but I'm not against this change.

Is this function commonly used? It is not called in the Python stdlib nor in Python test suite.

Reply:
It's called in site.py:

https://github.com/python/cpython/blob/fcbe0cb04d35189401c0c880ebfb4311e952d776/Lib/site.py#L457-L464

I'm not sure if it's commonly called by users in general.